### ISSUE TITLE
utf-16le mistaken for binary

### ISSUE THREAD

**Issue by mcandre** [djberg96/ptools#6]
ptools mistakenly treats some Unicode files as binary.

Example:

```
$ echo "test" > test.ascii
$ file -I test.ascii 
test.ascii: text/plain; charset=us-ascii
$ irb
> require 'ptools'
 => true 
> File.binary?('test.ascii')
 => false 
> exit

$ iconv -f ascii -t utf-16 test.ascii > test.utf-16
$ file -i test.utf-16 
test.utf-16: text/plain; charset=utf-16le
$ irb
> require 'ptools'
 => true 
> File.binary?('test.utf-16')
 => true 
> exit
```

System:

```
$ specs gem:ptools ruby file os
Specs:

specs 0.11
https://github.com/mcandre/specs#readme

gem list | grep ptools
ptools (1.2.6)

bundle --version
Bundler version 1.6.2

gem --version
2.2.2

ruby --version
ruby 2.0.0p481 (2014-05-08 revision 45883) [x86_64-linux]

file --version
file-5.14
magic file from /etc/magic:/usr/share/misc/magic

lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.1 LTS
Release:    14.04
Codename:   trusty
```

**Comment by djberg96** [comment]
@mcandre How's it look? https://github.com/djberg96/ptools/pull/35

**Comment by djberg96** [comment]
Fixed in 1.3.6, which was released today.
